tests/foreman/maintain/test_upgrade.py

```diff
@@ -98,8 +98,14 @@ def test_positive_repositories_validate(sat_maintain):
 @pytest.mark.parametrize(
     'custom_host',
     [
-        {'deploy_rhel_version': '8', 'deploy_flavor': 'satqe-ssd.disk.xxxl'},
-        {'deploy_rhel_version': '8', 'deploy_flavor': 'satqe-ssd.standard.std'},
+        {
+            'deploy_rhel_version': settings.server.version.rhel_version,
```

Comment:
Again, consider using `conf/content_host.yaml` here and in the following code (I'm not commenting on everything separately)

Reply:
See above